build.gradle

```diff
@@ -272,8 +272,12 @@ task startStoragesInDocker() {
     outputs.upToDateWhen { false }
 }
 
+task generateScmSourceJson(type: Exec) {
+    commandLine "scm-source", "-f", "scm-source.json"
```

Comment:
I think we should only generate this `scm-source.json` when building images that we push to pierone. Those generated by travis just to run tests do not require this.